<launch>
    <!-- Example finding 3D poses of the objects detected -->
    <!-- $ roslaunch kinect2_bridge kinect2_bridge.launch publish_tf:=true -->

    <!-- Which image resolution: sd, qhd, hd -->
    <arg name="resolution" default="qhd" />
<!--     <node name="loc_in_cam_node" pkg="me5413_world" type="loc_in_cam.py"  output="screen" /> -->
<!--     <node name="get_pose_node" pkg="me5413_world" type="get_pose.py" output="screen" /> -->
    <node name="find_object_3d" pkg="find_object_2d" type="find_object_2d">
        <param name="gui" value="true" type="bool"/>
        <param name="settings_path" value="~/.ros/find_object_2d.ini" type="str"/>
        <param name="subscribe_depth" value="true" type="bool"/>
        <param name="objects_path" value="$(find me5413_world)/media/template_d435" type="str"/>
        <param name="object_prefix" value="object" type="str"/>

        <remap from="rgb/image_rect_color" to="/camera_front/color/image_raw"/>
        <remap from="depth_registered/image_raw" to="/camera_front/depth/image_raw"/>
        <remap from="depth_registered/camera_info" to="/camera_front/depth/camera_info"/>
    </node>
    <node name="get_pose_node" pkg="me5413_world" type="send_goal.py"  output="screen"/>
</launch>
